### PR TITLE
 2foc 3.36, mc4plus 3.95, mc2plus 3.74, amc 2.8, 2foc-special 3.136

### PR DESCRIPTION
**Pull request by valegagge**
This PR is related to https://github.com/robotology/icub-firmware/pull/503 and https://github.com/robotology/icub-main/pull/972.

The 2foc-special is useful for AMI demos when they don't need the overheating fault check.

cc @MSECode

### CODE REVIEW

**Comment by S-Dafarra** [comment]
Hi @valegagge, my two cents would be that it is better not to have duplicated 2foc FW. It can be difficult to maintain both, and I am afraid that in a not-so-far future we will simply forget what ``special`` means here.

During the last demos, we simply disabled the temperature sensors since it is:
- faster
- easier to switch back
- more evident that there has been a modification.

The only time in which we used the special FW, we burnt down a motor as soon as we got home, and during the demo it turned out it was not useful to check the temperature.

In short, I would suggest to simply drop that FW.

**Comment by valegagge** [comment]
> Hi @valegagge, my two cents would be that it is better not to have duplicated 2foc FW. It can be difficult to maintain both, and I am afraid that in a not-so-far future we will simply forget what `special` means here.
> 
> During the last demos, we simply disabled the temperature sensors since it is:
> 
> * faster
> * easier to switch back
> * more evident that there has been a modification.
> 
> The only time in which we used the special FW, we burnt down a motor as soon as we got home, and during the demo it turned out it was not useful to check the temperature.
> 
> In short, I would suggest to simply drop that FW.

Hi @S-Dafarra , 
I prepared the 2foc-special fw because we agreed to provide you a fw for the demo. Do you remember that in the last PI we also prepared a script to install that fw in user-friendly way?

Of course if you no longer use it I'm very happy to remove from the repo for all the reasons you mentioned! 😄 

After the merge of this PR, I'm going to purge the repo from all references to this FW. (From script also)

**Review by marcoaccame — CHANGES_REQUESTED** [review]
Hi @valegagge, pls remember to change the [firmware.info.xml](https://github.com/robotology/icub-firmware-build/blob/devel/info/firmware.info.xml) file and then we can merge.

**Comment by valegagge** [comment]
> Hi @valegagge, pls remember to change the [firmware.info.xml](https://github.com/robotology/icub-firmware-build/blob/devel/info/firmware.info.xml) file and then we can merge.

done here: https://github.com/valegagge/icub-firmware-build/commit/bd997c2a007675f9569f4ad3fac5bd4dd48ccf4a

thanks!!!

cc @marcoaccame

**Review by marcoaccame — APPROVED** [review]
ok, thanks
